resources/META-INF/plugin.xml.template

```diff
@@ -8,20 +8,35 @@
 
   <category>Custom Languages</category>
 
-  <version>18.3</version>
+  <version>18.4</version>
```

Comment:
18.5 or 19?

Reply:
18.4, to match the version currently in `release_18`

Reply:
I'll push out a 19.0 once this lands.